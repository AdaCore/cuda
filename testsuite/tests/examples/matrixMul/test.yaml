driver: examples
input_directory: "0_Introduction/matrixMul"
