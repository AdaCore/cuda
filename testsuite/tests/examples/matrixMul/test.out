[Matrix Multiply Using CUDA] - Starting...
MatrixA (320, 320), MatrixB (640, 320)
Computing result using CUDA Kernel...
done
Computing result using Iteration...
done
Checking computed result for correctness: 
Correct cuda: 204800, errors: 0
Correct iter: 204800, errors: 0
Result = PASS
